src/components/Speaker.astro

```diff
@@ -31,9 +31,9 @@ const { speaker, index } = Astro.props;
   <div
     class="flex flex-col align-middle justify-center gap-1 md:gap-3 py-[0px] text-center lg:text-left"
   >
-    <h3 transition:name={`name-speaker-${index}`} class="text-2xl font-bold">
```

Comment:
this is supposed to be a heading, is there any reason for this change? 
![image](https://github.com/user-attachments/assets/c8e6888f-714d-4fba-9932-d5b35f735dc9)

Reply:
Although it is a strong text, I'm not sure if it should be a heading as it makes the web headings map a bit confusing or hard to traverse with each person count as a heading:
![image](https://github.com/user-attachments/assets/ca9ea428-f8ca-461a-93bc-efef2038c2ed)

Reply:
It's not a paragraph, though

Reply:
Left as as heading name at [b0df69b0ec652dd5f866913fc00ffc4144cddebc](https://github.com/biznaga-fest/web/pull/8/commits/b0df69b0ec652dd5f866913fc00ffc4144cddebc)